src/Tools/dotnet-monitor/Strings.resx

```diff
@@ -133,6 +133,10 @@
     <value>In 'Listen' mode, the diagnostic port endpoint name must be specified.</value>
     <comment>Gets a string similar to "In 'Listen' mode, the diagnostic port endpoint name must be specified.".</comment>
   </data>
+  <data name="ErrorMessage_DiagnosticPortNotInListenModeForCollectionRules" xml:space="preserve">
+    <value>When using collection rules, the diagnostic port must be in 'Listen' mode and the diagnostic port endpoint name must be specified. To use collection rules, please restart dotnet monitor with the updated configuration.</value>
```

Comment:
```suggestion
    <value>When using collection rules, the diagnostic port must be in 'Listen' mode. Please correct the configuration and restart dotnet monitor.</value>
```

More concise:
- We don't need to explain the full valid condition for using `listen` mode. Saying that the mode is required should be sufficient in order to spur the user to figure out what 'listen' mode is (presumably by looking at documentation) and how to configure it.
- I changed the second sentence around to imply order: first change the configuration, then restart dotnet monitor.